src/mapOfPeople.js

```diff
@@ -40,7 +40,13 @@
  * @return {Map}
  */
 function mapOfPeople(people) {
-  // write code here
+  const map = new Map();
+
+  people.forEach(function(item, i) {
```

Comment:
rewrite as arrow function and rename `item` according to name of the array

Reply:
done